### DISCUSSION TITLE
Multiple documentation sources 😟

### DISCUSSION THREAD

**Discussion by jerone** [vmware-clarity/ng-clarity#194] [Q&A] [answered]
I really don't understand the documentation sources anymore. For Angular Clarity there are multiple sites available:

* https://clarity.design - This seems to be the documentation for Core, with some information about Angular.
* https://angular.clarity.design - This seems to be dedicated to Angular components. But selecting the latest version (13) always redirects back to version 12.
* https://storybook.clarity.design - Also some information about Angular.

Every documentation site seems to contain different information, without **one** single source of truth.

Can someone help me clarify.

**Comment by ashleyryan** [comment]
You’re right, the state of our documentation is a mess right now. 

The best source of truth for Angular is angular.clarity.design. On clarity.design, we’ve removed most of the mention of angular components and recommended readers visit angular.clarity.design. 

Storybook.clarity.design only mentions Angular in the context of the Core components - so the @cds/angular package. If you’re using @clr/angular, the Angular subdomain is your best source of truth.

**Comment by bbogdanov** [comment]
Hi @jerone,

As as dev user I find myself using https://storybook.clarity.design/ and https://angular.clarity.design/ + https://github.com/vmware-clarity/ng-clarity/tree/main/projects/demo/src/app . The `https://clarity.design` helps me only about searching for icon shapes.

**Comment by 0xMurage** [comment]
https://angular.clarity.design now redirects to https://clarity.design.

As a beginner a beginner, everything is confusing.

**Comment by ashleyryan** [comment]
There was another change in the website 2 weeks ago after I left my original comment. As far as I'm aware, we're done moving things around.

clarity.design is the Angular & clr/ui documentation
core.clarity.design is the next-generation web-component and CSS Variable implementation of the design system.

They both have corresponding storybooks - storybook.clarity.design is for Angular components, storybook.core.clarity.design is for Core components

**Reply by jerone** [reply]
I noticed the change also. It looks like https://angular.clarity.design is now redirecting back to https://clarity.design.

Only the version selector still is broken there. Version 5 does not exists. Version 13 is absent. Version 12 is active, while the latest is 13 for a few months now.

**Reply by 0xMurage** [reply]
> core.clarity.design is the next-generation web-component and CSS Variable implementation of the design system.

I wish there was banner on the docs and repositories screaming that we have `@clr/core` vs `@cds/core` and `@clr/angular` vs `@cds/angular` in regards to what a beginner should use and contribute to.
 As a beginner, it hasn't been the best experience. It felt like mental torture. not something I would wish for newcomers.

**Reply by kevinbuhmann** [reply]
I do not disagree, and I wish we had the resources to make it better right now. We are working on hiring someone specifically to work on the Clarity websites.

If it helps, the `@clr` packages are here in the `ng-clarity` repo and the `@cds` packages are over in the `core` repo. (I am pretty sure the `@clr/core` package was replaced by `@cds/core` and is no longer updated.)

**Reply by 0xMurage** [reply]
**I was using some old docs that were still referencing @clr/core.**

I will give `core` packages a try and see what/how I could contribute to the community to reduce the confusion i.e. make it simple for beginners to get started

**Reply by juddkirby** [reply]
@kevinbuhmann how can we help pick up some of the low-hanging fruit in the meantime?  An example I just found is on the [contributing docs](https://github.com/vmware-clarity/ng-clarity/blob/main/docs/CONTRIBUTING_DOCUMENTATION.md#contributing-to-clarity-documentation)

```
The Clarity documentation is located at [angular.clarity.design](https://angular.clarity.design/) 
```

should now be

```
The Clarity documentation is located at [clarity.design](https://clarity.design/) 
```

I'm happy to put in PRs, but wanted to check in first to see what the process for minor documentation changes like this might look like.

Thanks!

**Reply by kevinbuhmann** [reply]
Feel free to open PRs against the docs!

**Comment by yusufozturk** [comment]
@ashleyryan thanks for the information. Could you also check this one:
https://github.com/vmware-clarity/ng-clarity/discussions/223#discussioncomment-3175578

**Comment by ad9999** [comment]
Can we have the link to the core storybook (as well as angular) somewhere on top-level clarity webpages ? I had to dig all the way in the comments to find out. 

storybook.core.clarity.design

**Reply by jerone** [reply]
Yeah, I'm still waiting for an official overview what site to use for what purpose, that I can accept as an answer.

**Comment by ashleyryan** [comment, marked as answer]
Links to storybook are on both sites. 

- For the Angular components (https://clarity.design) when you're in the documentation view, there's a link at the bottom of the left nav to the NG Storybook and to the Clarity Core website
- For Clarity Core web components (https://core.clarity.design), there is a link to Core Storybook in the left nav.

Yes, the two sites (or 4 really, when you factor in Storybook) is confusing, but we don't have the resources at the moment to prioritize investing much time into the websites.

**Reply by ad9999** [reply]
Thanks @ashleyryan for your prompt response. 

I can see the link now. Is it possible to add the link on the following page at the top of table?

https://clarity.design/documentation

**Comment by Rebolon** [comment]
And so... must we use @clr/angular (which seems to require @clr/ui) or @cds/angular (which doesn't have this dependancy) ? that's not easy because i started with clr/angular

**Reply by kevinbuhmann** [reply]
The `@clr/angular` package has our Angular components, and the `@cds/angular` package has Angular wrappers for the Clarity Core web components.
